libs/api-provider-environment/src/store/index.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
What's with all these?

Reply:
This applies to the SSR (Server-side Rendering) application, specifically in our case, Next.js. It helps identify the client-side components from the server-side components (you can read more about it [here](https://react.dev/reference/react/use-client)). We need to begin implementing this directive in our libraries that are used by SSR applications.

Other libraries adopting the Next.js client and server components architecture include:

- [Radix UI](https://github.com/radix-ui/themes/blob/main/packages/radix-ui-themes/src/theme.tsx#L1)
- [Material UI](https://github.com/mui/material-ui/blob/3bada36a7741e17ce0e1bca9de911602daa4b6f8/packages/mui-icons-material/lib/Tab.js#L2)